sale_import_base/tests/common_sale_order_import.py

```diff
@@ -0,0 +1,202 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from copy import deepcopy
+
+from odoo.addons.component.tests.common import SavepointComponentCase
+from odoo.addons.datamodel.tests.common import SavepointDatamodelCase
+from odoo.addons.sale.tests.test_sale_common import TestCommonSaleNoChart
+
+
+class SaleImportCase(
+    SavepointDatamodelCase, TestCommonSaleNoChart, SavepointComponentCase
+):
+    @classmethod
+    def setUpClass(cls):
+        super(SaleImportCase, cls).setUpClass()
+        cls.setUpClassicProducts()
+        cls.setUpAddresses()
+        cls.setUpLines()
+        cls.setUpImport()
+        cls.setUpTaxes()
+        cls.setUpFpos()
+        cls.setUpPayment()
+        cls.setUpMisc()
+
+    @classmethod
+    def setUpAddresses(cls):
+        cls.addr_customer_example = {
+            "name": "Thomas Jean",
+            "street": "1 rue de Jean",
+            "street2": "bis",
+            "zip": "69100",
+            "city": "Lyon",
+            "email": "thomasjean@gmail.com",
```

Comment:
Use a fake email like `thomasjean@example.org` the mail box of the real thomasjean@gmail.com will thank you ;)